### ISSUE TITLE
GraphQL failing to retrieve image info

### ISSUE THREAD

**Issue by dsilvam** [dotCMS/core#19449]
query

```{
  page(url: "/store/products/eagles-nest-outfitters-doublenest-hammock-patriot") {
    title
    urlContentMap{
      ... on Product{
      title
        tags
        urlTitle
        productNumber
        image{
          idPath
        }
      }
    }
    layout{
      body{
        rows{
          columns{
            containers{
              identifier
            }
          }
        }
      }
    }
  }  
}```

Throws 500 internal server error

**Comment by victoralfaro-dotcms** [comment]
PR: https://github.com/dotCMS/core/pull/19468

**Comment by dsilvam** [comment]
PR: https://github.com/dotCMS/core/pull/19481

**Comment by victoralfaro-dotcms** [comment]
Returned JSON:
```
{
  "data": {
    "page": {
      "title": "Product Detail",
      "urlContentMap": {
        "title": "Eagles Nest Outfitters DoubleNest Hammock Patriot",
        "tags": [
          "hiking",
          "camping"
        ],
        "urlTitle": "eagles-nest-outfitters-doublenest-hammock-patriot",
        "productNumber": "9315054521506-1280000",
        "image": {
          "idPath": "/dA/46e52dc2e7/image/hammock-1.jpg?language_id=1"
        }
      },
      "layout": {
        "body": {
          "rows": [
            {
              "columns": [
                {
                  "containers": [
                    {
                      "identifier": "//demo.dotcms.com/application/containers/default/"
                    }
                  ]
                }
              ]
            }
          ]
        }
      }
    }
  }
}
```

**Comment by bryanboza** [comment]
Fixed, tested on release-20.11 // Postgres // FF

**Comment by jcastro-dotcms** [comment]
The official code changes have been included in the dotCMS 5.3.8.4 LTS release only. The 5.2.8.4 LTS release is several commits behind, which made the code back-port not possible.
